browser/src/sass/modules/_form.scss

```diff
@@ -0,0 +1,36 @@
+.form {
+  @import url("https://fonts.googleapis.com/css?family=Roboto");
+  font-family: "Roboto", sans-serif;
+  $light-grey: #f5f9fa;
```

Comment:
toss color variables in `utilities/_config.scss` and just import that partial. I'll add a comment in there to indicate that it's for variables.